includes/class-wc-customizer-settings.php

```diff
@@ -431,6 +431,20 @@ public function get_settings() {
 
 					array( 'type' => 'sectionend' ),
 
+					array(
+						'title' => __( 'Cart Button', 'woocommerce-customizer' ),
```

Comment:
Should we change to Cart Page?  Because this button will be displayed on cart page.